edge/pkg/common/modules/names.go

```diff
@@ -1,14 +1,18 @@
 package modules
 
 const (
-	// EdgedModuleName name
+	// EdgeHubModuleName is EdgeHub module name
+	EdgeHubModuleName = "websocket"
```

Comment:
It seems to conflict with #3085

Reply:
yes, when conflict, i'll rebase this PR.